django_project/kartoza_project/models/project.py

```diff
@@ -26,6 +30,22 @@ class Project(Orderable, Slugged, AdminThumbMixin):
         verbose_name='Project thumbnail'
     )
 
+    country = models.TextField(
```

Comment:
Please use charfield instead, we should only use TextField for description or any text without maximum length.